queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
This SQL query will find `mon` anywhere in between an animal name, we are looking for an animal name that **ends** with `mon`

- [ ] Kindly update the query